design/media_types.go

```diff
@@ -30,6 +30,7 @@ var workItem = a.MediaType("application/vnd.workitem+json", func() {
 	a.Description("A work item hold field values according to a given field type")
 	a.Attribute("id", d.String, "unique id per installation")
 	a.Attribute("version", d.Integer, "Version for optimistic concurrency control")
+	a.Attribute("ExecutionOrder", d.Number, "Order of the workitem in the list")
```

Comment:
@aslakknutsen yes, i realise that its a mistake. I am fixing it in my next commit. And as we discussed, execution_order should be placed like version. In response, it should be within fields, but in database it should be stored seperately.